pkg/virt-controller/watch/migration.go

```diff
@@ -1217,7 +1217,7 @@ func (c *MigrationController) handlePendingPodTimeout(migration *virtv1.VirtualM
 			"Migration target pod for VMI [%s/%s] is currently unschedulable.", vmi.Namespace, vmi.Name)
 		log.Log.Object(migration).Warningf("Migration target pod for VMI [%s/%s] is currently unschedulable.", vmi.Namespace, vmi.Name)
 		if secondsSpentPending >= unschedulableTimeout {
-			return c.deleteTimedOutTargetPod(migration, vmi, pod, "unschedulable pod timeout period exceeded")
+			return c.deleteTimedOutTargetPod(migration, vmi, pod, "unschedulable pod %s timeout period exceeded")
```

Comment:
Since d51d7d47b21c0b8c

Reply:
On line 1228, there is another call to `deleteTimedOutTargetPod` would mind fixing it too?

Reply:
> On line 1228, there is another call to `deleteTimedOutTargetPod` would mind fixing it too?

OK, I missed one.